MapboxMobileEvents/MMEPreferences.h

```diff
@@ -62,6 +62,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Host SDK Version
 @property (nullable, nonatomic, copy) NSString *legacyHostSDKVersion;
 
+/// Unique Identifier for the client
```

Comment:
Guess I can only add comments to changed lines…

Please rename this class `MMEConfigation` and group it with `MMEConfigurationProviding` and `NSUserDefault+MMEConfiguration` files

Reply:
Updated